build.sbt

```diff
@@ -281,14 +278,15 @@ lazy val docs = http4sProject("docs")
         case Some(s) =>
           val isMaster = git.gitCurrentBranch.value == "master"
           val isSnapshot = git.gitCurrentTags.value.map(git.gitTagToVersionNumber.value).flatten.isEmpty
-
+          val gitHeadCommit = git.gitHeadCommit.value
+          val v = version.value
```

Comment:
A dynamic task is an alternative here, but I confess to not yet fully understanding them.